api/EventType.json

```diff
@@ -0,0 +1,104 @@
+{
+  "api": {
+    "EventType": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/Events",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
```

Comment:
Can you put `webview_android` to the end of the support block? (see issue #398 and #1882)